addons/block_code/ui/block_canvas/block_canvas.gd

```diff
@@ -45,7 +47,7 @@ signal replace_block_code
 
 
 func _ready():
-	if not _open_scene_button.icon:
+	if not Util.node_is_part_of_edited_scene(self) and not _open_scene_button.icon:
```

Comment:
Nitpick - optimize the logic by moving the simple `null` check first.

Reply:
I think the null check is now redundant: it will always be null when _ready runs.

Reply:
I left the null check in, but swapped the condition around.